### DISCUSSION TITLE
Is it possible to just make a duplicate row from existing row?

### DISCUSSION THREAD

**Discussion by mahirchavda** [pocketbase/pocketbase#939] [Q&A] [answered]
When i add record into some of my collection. I would like to just make a copy of the exiting record and update only some fields  fields that way i don't need to fill same fields values again.  I didn't find a duplicate or copy button in the collection list view.

**Comment by ganigeorgiev** [comment, marked as answer]
Currently this is not possible from the UI but I'll add it in my todo and may consider it for a future version.

**Reply by mahirchavda** [reply]
Thank you
